SPEC.md

```diff
@@ -85,7 +91,7 @@ Success is indicated by a return code of zero and the following JSON printed to
 
 ```
 {
-  "cniVersion": "0.2.0",
+  "cniVersion": "0.3.0",
```

Comment:
Good questions.

The main reason I changed it is because I was concerned that plugin authors and runtimes may be building against the `master` branch, and might be surprised to see changes in behavior without corresponding spec version bumps.  Bumping is low-cost and errs on the side of over-communicative, which I like.

Conversely, if we don't want plugin authors to be building against the `master` branch, then I'd say we need to be doing numbered releases much more often.

And if we are going to tie these together, it may make sense to actually require the numbers to be equal.  But we have already have a `v0.3` tag.  So perhaps we should immediately bump all version references on master to `0.4.0-rc`?

I know this topic was raised at our last maintainers meeting.  I can't recall if we came to a decision.  Other @containernetworking/cni-maintainers have thoughts?